src/visualization/types/SingleStatWithLine/view.tsx

```diff
@@ -266,11 +270,20 @@ const SingleStatWithLine: FC<Props> = ({properties, result, timeRange}) => {
               cellType: 'single-stat',
             })
 
-            const formattedValue = formatStatValue(latestValue, {
-              decimalPlaces: properties.decimalPlaces,
-              prefix: properties.prefix,
-              suffix: properties.suffix,
+            const timeFormatter = getFormatter('time', {
+              timeZone: timeZone === 'Local' ? undefined : timeZone,
+              timeFormat: DEFAULT_TIME_FORMAT,
             })
+
+            const formattedValue =
+              result.table.getColumnType('_value') == 'time'
```

Comment:
it's a small thing, but it's better to use `===` in most cases. `==` should be reserved for when you want to communicate that casting may happen. 

```js
3 === '3' // false
'3' === '3' // true
3 == '3' // true
```

so in most cases, we want to use `===` and the only time we don't is when we explicitly want to allow casting, like when converting between values that might be represented as a string or a number. For example, in HTML forms numbers get wrapped in strings. If we compare a value in redux that's a number to the value in a form that is a string, we'll want to use `==`